test_tools/sigma_verify_rules.py

```diff
@@ -76,8 +118,24 @@ def run_verifier(rules_path, config_file_path):
                     continue
 
                 rule_file_path = os.path.join(dirpath, rule_filename)
-                parsed_rule = sigma_util.get_sigma_rule(
-                    rule_file_path, sigma_config)
+
+                block_because_csv = False
+                if rule_file_path in ignore_list:
+                    return_rules_with_problems.append(rule_file_path)
+                    block_because_csv = True
+
+                if block_because_csv:
+                    continue
+
+                try:
+                    parsed_rule = sigma_util.get_sigma_rule(
+                        rule_file_path, sigma_config)
+                # This except is to keep the unknown exceptions
+                # this function is made to catch them and document
+                # them the broad exception is needed
+                except Exception:# pylint: disable=broad-except
+                    return_rules_with_problems.append(rule_file_path)
```

Comment:
do we not want to log this as well?

Reply:
yes we can, I opt we keep it as debug because otherwise it might get really noisy and by default, the problematic rules are printed at the end of the run anyway, but if then someone enables verbose, they would get the full info, which is actually a good idea.